<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Songs</class>
 <widget class="QMainWindow" name="Songs">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1008</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Песни</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="twSongs">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>60</y>
      <width>491</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSelectAll_1">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>20</y>
      <width>97</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Выбрать все</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="teResult">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>808</width>
      <height>141</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnConnect_1">
    <property name="geometry">
     <rect>
      <x>371</x>
      <y>20</y>
      <width>123</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Присоединиться</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>331</width>
      <height>218</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0">
      <widget class="QLabel" name="lbName_4">
       <property name="text">
        <string>Название</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="leName_4"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lbAlbumName">
       <property name="text">
        <string>Альбом</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="leAlbum"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lbComposer">
       <property name="text">
        <string>Композитор</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="leComposer"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="lbSongwriter">
       <property name="text">
        <string>Автор текста</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="2">
      <widget class="QLineEdit" name="leSongwriter"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="lbBand_4">
       <property name="text">
        <string>Группа</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="leBand_4"/>
     </item>
     <item row="5" column="1">
      <widget class="QPushButton" name="btnAdd_4">
       <property name="text">
        <string>Добавить</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QPushButton" name="btnDel_4">
       <property name="text">
        <string>Удалить</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1008</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
